nvim/nvim.go

```diff
@@ -679,3 +695,12 @@ type Mapping struct {
 	// Mode specifies modes for which the mapping is defined.
 	Mode string `msgpack:"string"`
 }
+
+type Channel struct {
+	ID     int           `msgpack:"id,omitempty"`
+	Stream string        `msgpack:"stream,omitempty"`
+	Mode   string        `msgpack:"mode,omitempty"`
+	Pty    string        `msgpack:"pty,omitempty"`
+	Buffer string        `msgpack:"buffer,omitempty"`
+	Client []interface{} `msgpack:"client,omitempty"`
```

Comment:
Can a struct type be defined for Client?

Reply:
AFAIK, The `Client` struct is this:
https://github.com/neovim/neovim/blob/b42c80e561c15fd49112dc46a7de04dd061a3483/src/nvim/api/vim.c#L995-L1035
It's can convert to the struct.

But, `Channel` struct also has constant variables such as `stream` value should `[stdio, stderr, socket, job]`. Can I write additional type use `iota`?
https://github.com/neovim/neovim/blob/b42c80e561c15fd49112dc46a7de04dd061a3483/src/nvim/api/vim.c#L1065-L1087